### PR TITLE
Updating the spanish version from the newsletter

### PR DESCRIPTION
**Pull request by Morban15**

### CODE REVIEW

File: es/index.html [hunk 1]
```diff
@@ -32,6 +32,7 @@
 	<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
 	<meta name="msapplication-TileColor" content="#da532c">
 	<meta name="theme-color" content="#ffffff">
+	<meta itemprop="datePublished" content="2019-07">
```
Comment (misscs) [inline es/index.html:35]: Don't we also need copyright?

Comment (Morban15) [reply es/index.html:35]: Yes I added. 

copyrightYear

File: es/index.html [hunk 2]
```diff
@@ -38,6 +38,7 @@
 <body>
 	<nav>
 		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
+			<span class="newsletter-date">July-October, 2019</span>
```
Comment (misscs) [inline es/index.html:41]: Whoops! Lo queremos en español

File: es/index.html (hunk 1 above)
Comment (misscs) [reply es/index.html:35]: Ah whoops! I didn't have other commits when I reviewed it for some reason. My bad!

**Pull request merged**